src/development/packages-and-plugins/plugin-testing.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Plugins in Flutter tests
+short-title: Plugin tests
+description: Adding plugin as part of your Flutter tests.
+---
+
+Almost all [Flutter plugins][] have two parts:
+
+* Dart code, which provides the API your code calls.
+* Code written in a platform-specific (or "host") language,
+  such as Kotlin or Swift, which implements those APIs.
+
+In fact, the native (or host) language code distinguishes
+a plugin package from a standard package.
+
+[Flutter plugins]: {{site.url}}/development/packages-and-plugins/using-packages
+
+Building and registering the host portion of a plugin
+is part of the Flutter application build process,
+so plugins only work when your code is running
+in your application, such as with `flutter run`
+or when running [integration tests][].
+When running [Dart unit tests][] or
+[widget tests][], the host code isn’t available.
+If the code you are testing calls any plugins,
+this often results in errors like the following:
+
+```sh
```

Comment:
This is output, not shell code, so it shouldn't have a language. This is causing "for" to be colorized.

Reply:
I'd go with marking it `console` which I suspect wont colorize for. I think.

Reply:
done